dynamic/main.go

```diff
@@ -147,6 +149,35 @@ func initialSetup() (info.Provider, pfbridge.ProviderMetadata, func() error) {
 				return plugin.ParameterizeResponse{}, err
 			}
 
+			if args.Remote != nil {
+				fullDocs = args.Remote.Docs
+				if fullDocs {
+					// Write the upstream files at this version to a temporary directory
+					tmpDir, err := os.MkdirTemp("", "upstreamRepoDir")
+					if err != nil {
+						return plugin.ParameterizeResponse{}, err
+					}
+					versionWithPrefix := "v" + info.Version
```

Comment:
Instead of calling this `versionWithPrefix`, let's call it what we assume it to be: the tag at which we are downloading the provider.
```suggestion
					tag := "v" + info.Version
```